lib/flok.js

```diff
@@ -214,8 +205,12 @@ Flok.prototype.down = function down(opts, done) {
     }],
 
     executeMigrations: ['blockedMigrations', function (callback) {
-      var migration = filter(self.migrations, 'isDone').slice(0, 1);
-      self._executeMigrations(migration, '_down', callback);
+      var migrations = filter(self.migrations, 'isDone').slice(0, 1);
```

Comment:
`find` returns an element of an array.  `slice` returns an array with 0 or 1 elements.

And `migrations = [find(...)]` wouldn't work either because if it found nothing there'd be an elem with undefined.   `_executeMigrations` has to be called even if with an empty array (for state reasons).  But it can't be called with 1 elem of `undefined` so I'd need to find then check to see if it found anything and if not set an empty array.  The slice is hacky I admit, but it stops it getting too wordy.